### DISCUSSION TITLE
Tarifas Pix cobradas pelos PSPs

### DISCUSSION THREAD

**Discussion by grudnitzki** [bacen/pix-api#266] [Q&A]
Seria interessante ter uma tabela comparativa das tarifas de serviços relacionados ao Pix cobradas pelos bancos e pelos demais PSPs.

Sou correntista do Itaú. Segue minha contribuição para a tabela:

ITAÚ
=======
Cada transferência recebida pelo Pix será tarifada pelo banco em 1,45% do valor pago.
O valor mínimo da tarifa por transação é R$1,75 e o valor máximo é R$10,00
Referência: https://www.itau.com.br/content/dam/itau/empresas/tarifas/tabela-geral-pj.pdf

Peço que ajudem enviando as tarifas que tiverem acesso. Vou compilar todas em uma planilha. Pode ser útil para escolher onde receber seus Pix e apontar abusos como o do Itaú.

**Comment by rubenskuhl** [comment]
Gerencianet: 1.98%, limitado a R$ 7,90. QR-Code dinâmico gerado adiciona 7,90. Fonte: https://gerencianet.com.br/tarifas/
(atualizado em 13.08.2021, antes era 0.99% limitado a R$ 0,89)

**Reply by marcianobandeira** [reply]
Desatualizado

**Reply by rubenskuhl** [reply]
Atualizado agora, mas fica o alerta de que negociando tipicamente se consegue ao menos a tarifa anterior, às vezes até menos.

**Reply by felipegps** [reply]
Qual a taxa mínima?

**Reply by rubenskuhl** [reply]
> Qual a taxa mínima?

Nesse PSP a taxa mínima é de 1 centavo, para taxas não negociadas. Negociações tipicamente aumentam a mínima e diminuem a máxima ou estabelecem valor fixo.

**Comment by rubenskuhl** [comment]
Mercado Pago: 0.99%, o céu é o limite. Fonte: https://www.mercadopago.com.br/ajuda/custo-receber-pagamentos_220

**Comment by rubenskuhl** [comment]
Adyen: 12 centavos de dólar mais 50 centavos de Real. Fonte: https://www.adyen.com/pt_BR/metodos-pagamento/pix
(Obs.: Recebimento não imediato, prazo similar a cartão de débito, e em nome da Adyen, que não é um PSP)

**Comment by rubenskuhl** [comment]
PagHiper (também não PSP, pagamento vai para a conta deles): R$ 0,99. Tarifa não pública, aparece para quem tem conta em https://www.paghiper.com/painel/detalhes-da-conta/ .

**Comment by grudnitzki** [comment]
Obrigado @rubenskuhl 

Entendi que o Banco Central cobrará dos bancos **1 centavo** para cada **10 transações** no Pix. Daí o banco multiplica este custo **por mil** e cobra uma exorbitância do correntista, para manter a bolada que sempre faturou com DOC, TED e boleto.

Ainda tenho esperança que alguma fintech cobre zero pelo Pix e não participe deste verdadeiro conluio.

**Reply by arantesxyz** [reply]
Olá, tudo bem?

A cobrança de R$0.01 a cada 10 mensagens cobrada pelo BCB do PSP é referente ao ressarcimento dos custos relacionados a operação do BCB apenas.

O PSP ainda tem outros custos para manter sua própria operação, como a própria infraestrutura, a conexão com a RSFN, monitoramento, equipes de risco, fraude, compliance, contábil, auditoria, financeiro, TI, etc.

O PIX já é obrigatoriamente gratuito para pessoas físicas, então os PSPs já assumem esse "prejuízo", e cobrar de pessoa jurídica é a forma de bancar esses custos e é claro, lucrar. Também é válido pontuar, que alguns PSP's, cobram apenas tarifas para transações via API, o que normalmente indica receita para quem está utilizando.

Lembrando que no final, essa pessoa jurídica que está pagando a tarifa, ela está repassando esse custo para o usuário final dela, pois ao montar o modelo de negócio da empresa, todos os custos são considerados (ou deveriam).

Dado essas pontuações, não vejo uma justificativa válida para não cobrar tarifa para operações via API.

**Reply by rubenskuhl** [reply]
Apesar disso, há várias não cobrando tarifa para operações Pix via API, como os bancos cooperativos Sicredi e Sicoob e as IFs BS2 e C6. Pode ser estratégia de mercado do tipo "loss leader" para ganhar em outros lugares ou para habituar o cliente ao usar o produto, ou para debugar um produto novo... 

... o único problema que eu vejo hoje na tarifação do BACEN é exigir que envio de Pix por pessoa física via API não possa ser tarifado. O resultado disso é que os PSPs mais focados em compliance com as normas não oferecem esse endpoint para PF, só para PJ. 

(Edit: Inter estava citado como cobrando zero pelo uso da API Pix, mas isso deixa de se aplicar a partir de Março/2024)

**Reply by arantesxyz** [reply]
Eu honestamente não sou a favor do uso da API PIX por pessoas físicas, principalmente pelo alto risco de fraude. E honestamente não vejo necessidade, pelo menos não para a geração de cobranças ou o envio de pagamentos.

**Reply by rubenskuhl** [reply]
O problema é que ainda é comum no país a "eupresa" baseada no CPF. E não é ilegal exercer atividade empreendedora com o CPF... é uma má idéia para o empreendedor pois não protege o patrimônio pessoal de problemas da empresa, mas é uma prerrogativa do empreendedor fazer assim se preferir.

**Comment by rubenskuhl** [comment]
Custar 0.001 pagos ao BACEN não significa que custa só isso... há diversos custos bem reais. Então zero eu não gostaria de pagar, pq eu quero ser o cliente e não o produto... mas pagar pouco sim. ;-)

**Comment by grudnitzki** [comment]
@rubenskuhl  Cobrar zero nem sempre significa que você é o produto. No caso dos bancos, eles já faturam **MUITO** com os valores recebido nas contas correntes. Estes valores muitas vezes são mantidos em conta pelo correntistas e aplicados em investimentos do próprio banco. Por isto é provável que algumas fintechs cobrem zero pelo Pix recebido, sem limites mensais.

**Comment by ivent** [comment]
JUNO

VENDER E RECEBER 
Pix Cobrança - Receber boleto - Com QR estático - Grátis
Boleto Pix  - Receber boleto com - QR Dinâmico = R$ 2,90 ( taxa fixa )

Fonte: https://campanhas.juno.com.br/pix-api

**Reply by rubenskuhl** [reply]
> JUNO
> 
> VENDER E RECEBER
> Pix Cobrança - Receber boleto - Com QR estático - Grátis
> Boleto Pix - Receber boleto com - QR Dinâmico = R$ 2,90 ( taxa fixa )
> 
> Fonte: https://campanhas.juno.com.br/pix-api

Com o alerta de que no momento ela ainda não está disponível, vide https://dev.juno.com.br/api/v2#tag/O-que-e-o-Pix
`Estamos readequando a oferta da API Pix de acordo com a Resolução BCB n° 30 de 29/10/2020. Em breve disponibilizaremos a API Pix para a geração de cobranças, conforme especificação do Banco Central.`

**Reply by ivent** [reply]
rubenskuhl sim neste caso eu optei pela opção do Boleto Pix que já está disponível via api, a cobrança realmente eles ainda estão implementando obrigado pela informação.

**Comment by aelsonamaral** [comment]
alguem sabe informar as tarifas do banco do brasil, bradesco, sicredi?

**Reply by leandrofreirex** [reply]
é buscar tarifa pix {nome_do_banco}
https://banco.bradesco/assets/pessoajuridica/pdf/tarifas/vigencia/01092021/servicos-pix.pdf
https://www.bb.com.br/docs/pub/trf/tarifasPJ.pdf
https://www.sicredi.com.br/media/produtos/filer_public/2022/08/31/tabela_de_tarifas_-_outubro_2022_-_310822.pdf

**Comment by aelsonamaral** [comment]
o que eu nao entendi e o sicredi, 9 reais/ nao e na %? igual outros bancos?

Em qua., 28 de set. de 2022 às 21:26, Leandro Freire <
***@***.***> escreveu:

**Reply by rubenskuhl** [reply]
> o que eu nao entendi e o sicredi, 9 reais/ nao e na %? igual outros bancos? Em qua., 28 de set. de 2022 às 21:26, Leandro Freire < ***@***.***> escreveu:
> […](#)
> é buscar tarifa pix {nome_do_banco} https://banco.bradesco/assets/pessoajuridica/pdf/tarifas/vigencia/01092021/servicos-pix.pdf https://www.bb.com.br/docs/pub/trf/tarifasPJ.pdf https://www.sicredi.com.br/media/produtos/filer_public/2022/08/31/tabela_de_tarifas_-_outubro_2022_-_310822.pdf — Reply to this email directly, view it on GitHub <[#266 (reply in thread)](https://github.com/bacen/pix-api/discussions/266#discussioncomment-3757794)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A3KRMMLLQU4OGQEPYB4HR2LWATO27ANCNFSM4X3XVDLQ> . You are receiving this because you commented.Message ID: ***@***.***>

Na verdade, o que não faz nenhum sentido é tarifa percentual no Pix. Estabelecer % é uma tentativa do sistema de defender a receita de cartões. 

Agora, é bom lembrar que essas tarifas de lista são tarifas de referência... o que não está escrito nelas é quanto de fato o banco costuma cobrar dos clientes.

**Reply by leandrovip** [reply]
> > o que eu nao entendi e o sicredi, 9 reais/ nao e na %? igual outros bancos? Em qua., 28 de set. de 2022 às 21:26, Leandro Freire < _**@**_._**> escreveu:
> > […](#)
> > é buscar tarifa pix {nome_do_banco} https://banco.bradesco/assets/pessoajuridica/pdf/tarifas/vigencia/01092021/servicos-pix.pdf https://www.bb.com.br/docs/pub/trf/tarifasPJ.pdf https://www.sicredi.com.br/media/produtos/filer_public/2022/08/31/tabela_de_tarifas_-_outubro_2022_-_310822.pdf — Reply to this email directly, view it on GitHub <[#266 (reply in thread)](https://github.com/bacen/pix-api/discussions/266#discussioncomment-3757794)>, or unsubscribe https://github.com/notifications/unsubscribe-auth/A3KRMMLLQU4OGQEPYB4HR2LWATO27ANCNFSM4X3XVDLQ . You are receiving this because you commented.Message ID: _**@**_.**_>
> 
> Na verdade, o que não faz nenhum sentido é tarifa percentual no Pix. Estabelecer % é uma tentativa do sistema de defender a receita de cartões.
> 
> Agora, é bom lembrar que essas tarifas de lista são tarifas de referência... o que não está escrito nelas é quanto de fato o banco costuma cobrar dos clientes.

Acredito que tarifas por % faz mais sentido em determinados casos. Por exemplo, temos alguns clientes que possuem um bom fluxo de vendas, porém o ticket médio baixo (=< 15,00). Em pesquisas sobre tarifas para integrarmos o recebimento via pix, uma grande parte das empresas (psp e fintech) trabalham com um valor fixo por transação, de 0,50 a 3,00 (mínimo). Neste cenário seria inviável financeiramente. Ótimo para quem paga e custoso para quem recebe. 

Não acho que teria que ser grátis, mas para certos cenários como este, % seria melhor visando o custo-beneficio.  Faz sentido?

**Reply by rubenskuhl** [reply]
> > > o que eu nao entendi e o sicredi, 9 reais/ nao e na %? igual outros bancos? Em qua., 28 de set. de 2022 às 21:26, Leandro Freire < _**@**_._**> escreveu:
> > > […](#)
> > > é buscar tarifa pix {nome_do_banco} https://banco.bradesco/assets/pessoajuridica/pdf/tarifas/vigencia/01092021/servicos-pix.pdf https://www.bb.com.br/docs/pub/trf/tarifasPJ.pdf https://www.sicredi.com.br/media/produtos/filer_public/2022/08/31/tabela_de_tarifas_-_outubro_2022_-_310822.pdf — Reply to this email directly, view it on GitHub <[#266 (reply in thread)](https://github.com/bacen/pix-api/discussions/266#discussioncomment-3757794)>, or unsubscribe https://github.com/notifications/unsubscribe-auth/A3KRMMLLQU4OGQEPYB4HR2LWATO27ANCNFSM4X3XVDLQ . You are receiving this because you commented.Message ID: _**@**_.**_>
> > 
> > 
> > Na verdade, o que não faz nenhum sentido é tarifa percentual no Pix. Estabelecer % é uma tentativa do sistema de defender a receita de cartões.
> > Agora, é bom lembrar que essas tarifas de lista são tarifas de referência... o que não está escrito nelas é quanto de fato o banco costuma cobrar dos clientes.
> 
> Acredito que tarifas por % faz mais sentido em determinados casos. Por exemplo, temos alguns clientes que possuem um bom fluxo de vendas, porém o ticket médio baixo (=< 15,00). Em pesquisas sobre tarifas para integrarmos o recebimento via pix, uma grande parte das empresas (psp e fintech) trabalham com um valor fixo por transação, de 0,50 a 3,00 (mínimo). Neste cenário seria inviável financeiramente. Ótimo para quem paga e custoso para quem recebe.
> 
> Não acho que teria que ser grátis, mas para certos cenários como este, % seria melhor visando o custo-beneficio. Faz sentido?

Para ser percentual, significa que há subsídio cruzado em que os comerciantes de ticket maior subsidiam o que os de que ticket menor pagam. Então, não faz sentido percentual em nenhum caso. 

Se o valor de transação de 50 centavos a 3 reais é pesado, significa que o valor de transação menor mas fixo é que resolveria... se fosse 1 centavo, 10 centavos etc. E é isso que deveria ser buscado, e não uma distorção de mercado que fatalmente cria ineficiências. 

Por exemplo, no passado o sistema de telefonia fazia subsídio cruzado entre ligações locais e de longa distância... e a solução foi acabar com essas distorções, não só diminuí-las de valor. 

Notar que o custo base do Pix é R$ 0.001 pagos ao Banco Central, então valores bem baixos de transação conseguiriam remunerar o prestador de forma sustentável e lucrativa. Se não é isso que o mercado está oferecendo, é pq há outras forças em ação.

**Reply by giovanicmartins** [reply]
Emissão de cobrança instantânea pela fci-it.com.br R$ 0,45 por transação na liquidação. Sem limite de valor. Campanha de incentivo de monetização para DEVs e S_house.

**Comment by brubcruz** [comment]
Parece não atualizado a um tempo esse post.
Pelo que eu estou vendo, hoje o mais barato seria o OpenPIX com R$ 0,85 por transação ou 0.8%.
Tem uma documentação boa e opção de transferir o dinheiro direto para outra conta qualquer.

**Reply by rubenskuhl** [reply]
Acho que isso ficou desatualizado com a mudança de marca para Woovi, que agora é sempre 1%:
https://woovi.com/pricing/

De qualquer forma, os bancos cooperativos como SICREDI e SICOOB ainda estão com tarifas zeradas na API Pix, e nenhum número positivo é menor do que zero...

**Reply by brubcruz** [reply]
Pelo que me passaram realmente o preço é como eu falei:
https://openpix.com.br/pricing/

De qualquer forma, então atualmente podemos integrar os sistemas, emitir os pix/qrcode, receber as cobranças, transferir para outras contas bancarias, tudo isso sem custos pelo SICREDI ou SICOOB? Eu não sabia disso..
Atualmente utilizo o gerencianet/efi para emitir boletos e pix. Pretendo migrar para outro e parar de emitir boletos, ficando apenas com o PIX.

Obrigado pelas informações Rubens.

**Reply by rubenskuhl** [reply]
Vou avisar o pessoal de lá para vir aqui e confirmar.

**Comment by b2ubank** [comment]
Neste banco custa a partir de R$ 0,19 e tem API

www.b2ubank.com.br

![image](https://github.com/bacen/pix-api/assets/137004747/7dabcafd-70d5-4f9e-aff4-644a0168f796)

**Reply by rubenskuhl** [reply]
Faltou dizer no texto (na imagem já diz) que 19 centavos só se tiver 600 mil Pix por mês... o que é um número bem pequeno de empresas que geram. 

E faltou também dizer que a API é totalmente proprietária e que não é PSP (usa um PSP chamado Cartos). Mesmo não sendo um PSP, seria possível seguir a API padrão do Banco Central como são obrigados os PSPs, para facilidade dos clientes.

**Comment by giovanicmartins** [comment]
Faltou ainda dizer que cobram pelo setup.

Pra quem procura PIX e monetizar como parceiro, procura por @fcipay


Em seg., 19 de fev. de 2024, 22:46, Rubens Kuhl ***@***.***>
escreveu:

**Comment by b2ubank** [comment]
O B2U Bank não cobra por setup, acreditamos que os desenvolvedores precisam
ter mais de uma, duas, três, quatro opções de API de PIX para integrarem ao
sistema, do contrário corre o risco de ficar na mão de um só.

A descentralização é benéfica!



Em ter., 27 de fev. de 2024 às 00:12, Giovani Martins <
***@***.***> escreveu:

**Comment by otaviogrrd** [comment]
pagbank 1,89% por pagamento

https://faq.pagbank.com.br/duvida/o-que-e-e-como-funciona-a-cobranca-via-pix/1331

**Reply by felipegps** [reply]
Qual taxa mínima?

**Comment by otaviogrrd** [comment]
Banco Inter:
A partir do dia 01/03/2024, está sendo cobrada a tarifa no recebimento de cobranças geradas via API Pix. 
Foi enviado um comunicado por e-mail no dia 01/02/2024 informando o início dessa tarifação. 
A partir dos 30 primeiros recebimentos do mês, é cobrada uma tarifa de 0,9% da transação, com valor mínimo de R$ 0,10 e máximo de R$ 1,50

em resumo:
30 liquidações gratuitas, depois disso: 
se é liquidado um PIX de R$ 1 será cobrado R$0,10
se é liquidado um PIX de R$ 1000 será cobrado R$ 1,50

**Comment by otaviogrrd** [comment]
Banco BTG:
As APIs do BTG Pactual Empresas só podem ser consumidas com o Plano Avançado: R$ 200,00 /mês
200 transações ao mês (TED, Boleto liquidado e Liquidação de QR Code)
Transação excedente R$ 1,50

(consultor do banco informou que pode haver negociação dessa taxa excedente, mas nada garantido)

**Comment by giovanicmartins** [comment]
www.fcipay.com.br

Centavos por transação liquidada. Para parceiros tem fee.

https://linktr.ee/fcipay

Em qui., 13 de jun. de 2024, 14:17, Otávio Garrido Moraes <
***@***.***> escreveu:

**Comment by rafauel** [comment]
Por acaso alguém sabe me dizer como se tornar psp ou alguma forma de integrar minha empresa com Bacen para aceitar pix? Poder criar chave de pix etc? Estou criando uma empresa de pagamentos e queria poder oferecer recebimento de pix para os clientes, mais não sei como faz isso sem me vincular há uma psp ou adquirente

**Reply by rubenskuhl** [reply]
> Por acaso alguém sabe me dizer como se tornar psp ou alguma forma de integrar minha empresa com Bacen para aceitar pix? Poder criar chave de pix etc? Estou criando uma empresa de pagamentos e queria poder oferecer recebimento de pix para os clientes, mais não sei como faz isso sem me vincular há uma psp ou adquirente

https://www.bcb.gov.br/estabilidadefinanceira/instituicaopagamento

Você não precisa se vincular a um PSP ou adquirente, mas muito provavelmente precisará de um PSTI. 
https://www.bcb.gov.br/estabilidadefinanceira/comunicacaodados

Veja os PSTIs disponíveis para conexão. Alguns podem ter um serviço mais tipo "pegar pela mão" para consultoria nesse processo.

**Reply by rafauel** [reply]
Obrigado pela ajuda! Neste caso está me falando que posso fazer integração com o banco central certo? Apenas preciso me credenciar. ?

**Reply by rubenskuhl** [reply]
> Obrigado pela ajuda! Neste caso está me falando que posso fazer integração com o banco central certo? Apenas preciso me credenciar. ?

"Apenas" é um otimismo bastante grande... o caminho de autorização é árduo e com muitos requisitos pesados. Por isso que a maioria começa como operação junto a algum BaaS e só depois migra para credenciais próprias.

**Reply by rafauel** [reply]
Esses baas, você poderia citar algum no mercado para que eu possa ver taxas e regras ?? 

A minha ideia é ter essa integração com pix de uma forma que terei uma taxa menor do que usar um gatawey ou algo assim.

**Reply by rubenskuhl** [reply]
> Esses baas, você poderia citar algum no mercado para que eu possa ver taxas e regras ??

BTG, BV, Celcoin, Dock

> A minha ideia é ter essa integração com pix de uma forma que terei uma taxa menor do que usar um gatawey ou algo assim.

Não é por aí. Pelo contrário, as taxas são tipicamente mais altas, e há diversos custos fixos que você teria como ente regulado que vão ter pouco ganho de escala e prejudicar sua conta de chegada. 

O jeito de diminuir custos de gateway é procurar bem e negociar... sugiro a lista do #76 .

**Reply by rafauel** [reply]
Vamos lá, deixa eu explicar um pouco melhor: eu tenho um sisteminha de pagamento na qual estou implementando em alguns comerciantes, onde eles podem receber do cliente deles. Atualmente estou usando a integração com a pagar-me. Porém visando ter  um sistema mais robusto, eu pretendo liberar uma funcionalidade onde o cliente possa enviar pix para mim e também quando ele pedir algum saque, eu possa enviar para ele via pix. 

Então hoje com a pagar-me eu consigo receber via pix, com uma taxa a cima de 1%. Porém não consigo enviar para qualquer pessoa.

Por isso a necessidade de uma empresa na qual posso gerar pix de cobrança e cair na minha conta e pagamento pix na qual posso enviar para qualquer pessoa..

Comecei a estudar esse mundo pix agora e por isso talvez minhas perguntas seja iniciante de mais.

**Reply by pedrojobs13** [reply]
> PSTI

Você consegue integrar mas terá que se cadastrar no sisbacen -> https://www.bcb.gov.br/meubc/sisbacen

**Reply by b2ubank** [reply]
Vai gastar uns 300 mil reais para ser pix participante direto, é muita burocracia. A fila é grande, vai ter que esperar uns 2 anos. Tem outras opções no mercado, que não cobram %, mas valores fixos, por exemplo R$ 0,99 por transação.

**Comment by giovanicmartins** [comment]
Sobre Manda email com contato para ***@***.***

Em qua., 19 de jun. de 2024, 19:55, Rafael Antonio ***@***.***>
escreveu:

**Reply by rafauel** [reply]
Não entendi!

**Comment by jeeanribeiro** [comment]
Estou trabalhando em um jogo com microtransações de centavos (0,10) via PIX, qual seria a API mais vantajosa em termos de preço?

Dando uma pesquisada aqui encontrei o efí bank que parece não ter taxa mínima

**Reply by rubenskuhl** [reply]
> Estou trabalhando em um jogo com microtransações de centavos (0,10) via PIX, qual seria a API mais vantajosa em termos de preço?
> 
> Dando uma pesquisada aqui encontrei o efí bank que parece não ter taxa mínima


Para recebimento a Efí tem taxa mínima de 1 centavo. Para envio, taxa mínima de 50 centavos. 

Acho que acumular um pouco as transações seria melhor...

**Comment by victoranalista** [comment]
Hoje qual o banco que cobra as menores taxas pix? para pagamentos e recebimento via api em lote?

**Reply by rubenskuhl** [reply]
> Hoje qual o banco que cobra as menores taxas pix? para pagamentos e recebimento via api em lote?

O banco que estiver com taxa zero promocional, enquanto isso durar... Inter e Sicredi já foram esse banco, mas começaram a cobrar segundo relatos. Sicoob parece que ainda não está cobrando. 

Sicredi e Sicoob são um pouco mais difíceis de saber tarifas por serem grêmios de cooperativas... então as condições de cada cidade podem ser diferentes. Precisa perguntar na sua cidade. 

Mas toda taxa zero de hoje será a taxa não-zero de amanhã... então precisa estar sempre de malas prontas para mudar para outro prestador.

**Comment by abm1982** [comment]
Alguém já usou a Uzzypay? Eles estão com taxa de 0,80% por pix. Acho que não tem menor no mercado. Estou avaliando mudar do MP pra lá.

**Reply by rubenskuhl** [reply]
Pra volume isso é caro. Com volume se consegue menos de 10 centavos por Pix, independente de valor. 
Essa mania de taxas percentuais, idéia do citado MP, é uma doença que um dia precisaremos extirpar.

**Reply by abm1982** [reply]
Oi Rubens, meu negócio é novo e estou um pouco perdido. Atualmente meu site aceita pagamentos via pix pelo MP (0,99% por pix) e deixei a EFI programada de reserva (1,19% por pix).

Vocâ conhece alguma alternativa melhor API PIX?

**Reply by rubenskuhl** [reply]
Eu mantenho uma lista de PSPs com API Pix neste tópico: #76 

Mas com mais foco na aderência ao regulamento do que em tarifas... tem PSPs que temporariamente não cobram (quais é algo dinâmico), então pode ser uma opção para novos negócios até ter volume para negociar.

**Reply by abm1982** [reply]
muito obrigado

**Comment by yvesmariano** [comment]
AbacatePay cobra R$ 0,80 por transação https://www.abacatepay.com
Atualmente utilizando estrutura da Woovi

**Reply by rubenskuhl** [reply]
> AbacatePay cobra R$ 0,80 por transação https://www.abacatepay.com
> 
> * atualmente utilizando estrutura da Woovi

Acho que atualmente é estrutura da AsaaS, mas o preço é esse mesmo. (Ref. 07.07.2025)

**Comment by felipegps** [comment]
Alguém já conseguiu configurar C6 Bank? Pelo que parece é gratuito!

**Reply by rubenskuhl** [reply]
> Alguém já conseguiu configurar C6 Bank? Pelo que parece é gratuito!

Mas eles tem alguma forma para acessar que não seja via PayGo ? Pq PayGo além de não ser gratuito (mesmo que não haja custo adicional no C6), usa API fora do padrão.

**Comment by aelsonamaral** [comment]
https://payzu.com.br Payzu
Usa infra da celcoin (0,25 centavos) aqui tá sendo ok a 5 meses

Em seg., 7 de jul. de 2025 às 19:44, Rubens Kuhl ***@***.***>
escreveu:

**Comment by brubcruz** [comment]
Vou tentar fazer um panorama geral básico aqui baseado no que entendi até o momento, focado nos preços.
Alguém me corrija se eu estiver errado.

Pelo que estou entendendo, em geral os PSP "tradicionais"como o EFI tem cobrado em porcentagem, algo em torno de 1%.
O que pra min é caro e inviável. Outras parecidas seriam:
Itau, Santander, Inter, PagBank, pagar.me, etc.

Diversas outras continuam surgindo nesses últimos anos, como por exemplo:
openPix (woovi), abacatepay(woovi), Asaas, etc
Essas conseguem um preço em torno de R$ 0,80 por transação paga.
O que é um pouco mais viável, na minha opinião.

E pelo que vi, algumas mais recentes ainda como payzu e "push in pay", conseguem valor em trono de R$ 0,30 por transação paga.

Até onde sei, o SICREDI tem a API gratuita, mas não sei os tramites para abrir conta com eles.

Já integrei meu sistema com EFI e OpenPIX, sem grandes problemas até então.
Não me importo tanto se o PSP segue a "API PIX Padrão", pois creio que cada vez que se migra para uma nova, vai ter um certo nivel de desenvolvimento de qualquer forma.

**Reply by rubenskuhl** [reply]
Há uma grande diferença entre as tarifas públicas e as negociadas. A maior parte das tarifas públicas é em especificada em percentual, como você relatou na Efí... eu atribuo essa criação ao MercadoPago, que no começo do Pix era o único com percentual, que depois foi imitado. Mesmo a Woovi que você citou cobra ou 85 centavos fixo, ou 0.8% com mínimo de 50 centavos, máximo de R$5. 

Agora, todos esses tradicionais que você citou te dão preço fixo por transação se você tiver volume e negociar. Vou dar um exemplo de Itaú: sei de quem paga 10 centavos fixo por transação. Um exemplo de Efí: sei de quem preço fixo por transação, mas protegido por cláusula de confidencialidade então não vou citar qual seja ele. 

Dos bancos cooperativos como SICREDI e SICOOB, o que me relataram é que agora o SICREDI está cobrando, e o SICOOB ainda não. Vários prestadores começaram gratuito (ex: Inter) e depois mudaram para pago (como Inter e SICREDI estariam agora). 

Sobre a API Pix padrão, ela é a principal linha de defesa para você ter liberdade de escolha. Um nível baixo de desenvolvimento para uma eventual mudança, da ordem de dias, comparado a meses de desenvolvimento para uma nova integração, faz muita diferença nas decisões de um recebedor.

**Reply by brubcruz** [reply]
Obrigado pela resposta Rubens.

Concordo com voce e, vejo que o que vocês disse esta muito correto.

Sobre as tarifas publicas, acho interessante notar que essa negociação permanece a mesma desde anos atrás quando só haviam boletos como opção.
Então as empresas que tem volume conseguem preços bem menores e, com a chegada do pix isso não mudou.
Porém eu falo do ponto de vista de uma empresa pequena, como acredito que tenham varias por ai, e essas não tem acesso a esses preços diferenciados.

Do ponto de vista de preços, eu imaginava que com o PIX, as cobranças seriam barateadas muito mais rápido do que de fato vem acontecendo, embora, em comparação com boletos melhorou bem.
Agora chegando em torno de R$ 0.30 parece finalmente um preço bem competitivo.

E mesma coisa sobre a API padrão. Eu tinha EFI que é padrão, e depois implementei o woovi em menos de uma semana.
Mas concordo com você que para empresas maiores, que são mais burocráticas e complexas, esse desenvolvimento pode demorar bem mais, portanto é de fato muito importante ter essa facilidade da API padrão.

**Reply by albertobraschi** [reply]
> Há uma grande diferença entre as tarifas públicas e as negociadas. A maior parte das tarifas públicas é em especificada em percentual, como você relatou na Efí... eu atribuo essa criação ao MercadoPago, que no começo do Pix era o único com percentual, que depois foi imitado. Mesmo a Woovi que você citou cobra ou 85 centavos fixo, ou 0.8% com mínimo de 50 centavos, máximo de R$5.
> 
> Agora, todos esses tradicionais que você citou te dão preço fixo por transação se você tiver volume e negociar. Vou dar um exemplo de Itaú: sei de quem paga 10 centavos fixo por transação. Um exemplo de Efí: sei de quem preço fixo por transação, mas protegido por cláusula de confidencialidade então não vou citar qual seja ele.
> 
> Dos bancos cooperativos como SICREDI e SICOOB, o que me relataram é que agora o SICREDI está cobrando, e o SICOOB ainda não. Vários prestadores começaram gratuito (ex: Inter) e depois mudaram para pago (como Inter e SICREDI estariam agora).
> 
> Sobre a API Pix padrão, ela é a principal linha de defesa para você ter liberdade de escolha. Um nível baixo de desenvolvimento para uma eventual mudança, da ordem de dias, comparado a meses de desenvolvimento para uma nova integração, faz muita diferença nas decisões de um recebedor.

O banco inter ate onde eu sei, eles nao cobram de PJ nas vendas  ate certo limite, depois do limite tem alguma tarifa
O C6 nao cobra pelo mesmo serviço, e nao tem limites
O Sicoob exige certificado A1, e no sandbox voce nao consegue testar completamente, se é que consegue testar alguma coisa; porem, o Sicoob nao cobra o serviço

**Reply by webmidiacrm-eng** [reply]
> Obrigado pela resposta Rubens.
> 
> Concordo com voce e, vejo que o que vocês disse esta muito correto.
> 
> Sobre as tarifas publicas, acho interessante notar que essa negociação permanece a mesma desde anos atrás quando só haviam boletos como opção. Então as empresas que tem volume conseguem preços bem menores e, com a chegada do pix isso não mudou. Porém eu falo do ponto de vista de uma empresa pequena, como acredito que tenham varias por ai, e essas não tem acesso a esses preços diferenciados.
> 
> Do ponto de vista de preços, eu imaginava que com o PIX, as cobranças seriam barateadas muito mais rápido do que de fato vem acontecendo, embora, em comparação com boletos melhorou bem. Agora chegando em torno de R$ 0.30 parece finalmente um preço bem competitivo.
> 
> E mesma coisa sobre a API padrão. Eu tinha EFI que é padrão, e depois implementei o woovi em menos de uma semana. Mas concordo com você que para empresas maiores, que são mais burocráticas e complexas, esse desenvolvimento pode demorar bem mais, portanto é de fato muito importante ter essa facilidade da API padrão.

Em relação a essas taxas, minha realidade hj é Faturamento mensal de 300 mil, tendo cerca de 1500 transações PIX por mês, a ideia seria justamente agilizar o processo com baixa automática no Pix dentro do ERP. Estou dentro do patamar para negociar essas taxas?

**Comment by albertobraschi** [comment]
O banco digital do itau, itau emps, parece q nao vai cobrar o pix

**Reply by yvesmariano** [reply]
Mas tem API Pix?

**Comment by webmidiacrm-eng** [comment]
Em relação a essas taxas, minha realidade hj é Faturamento mensal de 300 mil, tendo cerca de 1500 transações PIX por mês, a ideia seria justamente agilizar o processo com baixa automática no Pix dentro do ERP. Estou dentro do patamar para negociar essas taxas?

**Reply by yvesmariano** [reply]
Acredito que sim, @webmidiacrm-eng. A partir de 1000 transações ou 100 mil transacionado mensalmente os gateways já estão dispostos a negociar. Se o teu gateway não tiver disposto pode procurar outro que vai lhe receber muito bem.

**Comment by abm1982** [comment]
Bom dia! Alguém já teve problema com MED no Mercado Pago? Não fui notificado e hoje encontrei um estorno de uma venda que fiz 2 meses atrás e foi contestada. O cara usou o serviço e depois abriu MED.

**Reply by yvesmariano** [reply]
> Bom dia! Alguém já teve problema com MED no Mercado Pago? Não fui notificado e hoje encontrei um estorno de uma venda que fiz 2 meses atrás e foi contestada. O cara usou o serviço e depois abriu MED.

Agora que os bancos são obrigados a permitir que o cliente inicie o MED sem necessidade de passar por um atendimento humano explodiu o número de MEDs. Minha dica é fuja do Mercado Pago, não precisa de muito para cancelarem tua conta.

**Reply by abm1982** [reply]
O que mais fiquei bravo com o MP foi que não me avisaram. Descobri por acaso olhando o extrato e vi um estorno. Tentei migrar para a EFI mas não consegui implantar 100% nem com ajuda de GPT. Aquele negócio de p12 é meio complicado pra quem não entende. Estou olhando a Woovi e Pagbank. 

O que acha? Mais alguma sugestão para pagamentos pix?

**Reply by luizpaulolppa** [reply]
@abm1982 tem a Trio Pagamentos, pode ser que te ajude também

**Reply by abm1982** [reply]
Ah legal Luiz, vou dar uma olhada! Estou olhando a PicPay também...taxa 0,99% igual do MP

**Comment by abm1982** [comment]
Pessoal, atualmente estou na Woovi por 0,80% por pix, porém consegui da Pagar.me (Stone) 0,78%. 

Não é muita diferença, mas o que me animou na Pagar.me é que eles estão desenvolvendo um app para a conta, a Woovi não tem.

**Comment by sambattsd** [comment]
Aqui na PayZu oferecemos dois planos de PIX: R$ 0,50 por transação (cash-in ou cash-out), ou 0,50% no cash-in + R$ 0,50 no cash-out, com mínimo de R$ 0,20 e máximo de R$ 5,00 no cash-in. Tudo com integrações 100% via API, conta em sua própria titularidade e a solidez do FitBank por trás. Não temos mensalidade, consumo mínimo ou qualquer tipo de surpresa, você paga apenas pelo uso. E, sabendo como são os dias de hoje, qualquer contestação via MED você mesmo responde, afinal há muita gente agindo de má-fé. Para abrir sua conta, acesse https://abrirconta.payzu.com.br/ ou fale no WhatsApp: +55 11 5128-9296.

**O único porém é que atendemos somente PJ.**
